platform/kube/client.go

```diff
@@ -406,3 +406,29 @@ func (cl *Client) GetTLSSecret(uri string) (*model.TLSSecret, error) {
 		PrivateKey:  key,
 	}, nil
 }
+
+// Request sends requests through the Kubernetes apiserver proxy to
+// the a Kubernetes service.
+// (see https://kubernetes.io/docs/concepts/cluster-administration/access-cluster/#discovering-builtin-services)
+func (cl *Client) Request(namespace, service, method, path string, inBody []byte) ([]byte, error) {
+	absPath := fmt.Sprintf("api/v1/namespaces/%s/services/%s/proxy/%s/%s",
+		namespace, service, IstioResourceVersion, path)
+	var status int
+	outBody, err := cl.dyn.Verb(method).
```

Comment:
is `dyn` the right HTTP client? it's configured to use istio TPRs which is not necessary for this client? maybe use `cl.client`?